src/DevChatter.Bot.Core/Games/DealNoDeal/DealNoDealGame.cs

```diff
@@ -0,0 +1,342 @@
+using DevChatter.Bot.Core.Automation;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Events;
+using DevChatter.Bot.Core.Extensions;
+using DevChatter.Bot.Core.Systems.Chat;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DevChatter.Bot.Core.Games.DealNoDeal
+{
+    public class DealNoDealGame : IGame
+    {
+        public bool IsRunning { get; private set; }
+
+        //Configuration
+        public const UserRole ROLE_REQUIRED_TO_START = UserRole.Everyone;
+        public const int SECONDS_TO_CHOOSE_BOXES = 30;
+        public const int TOKENS_TO_PLAY = 25;
+        public const int MIN_PLAYABLE_BOXES = 5;
```

Comment:
All public members should be pascal cased according to https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers. Still a taste thing though.